app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt

```diff
@@ -95,6 +97,14 @@ class SellPayloadPresenter(
         }
     }
 
+    override fun onStart() {
+        launch { if (needCheckForSellLock) checkForSellLock() }
+    }
+
+    override fun onStop() {
+        needCheckForSellLock = true
```

Comment:
I cannot get, why are you changing this value on stop. 
What is the idea?

Reply:
`checkForSellLock()`call on `attach` `org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt:65` and have comment `// call order is important!`
also we need to call `checkForSellLock()` only after return from moonpay webView for `navigateToSellLock` `org/p2p/wallet/sell/ui/payload/SellPayloadPresenter.kt:89`

so if fragment livecycle only onStop()-> onStart() need to call `checkForSellLock`
if fragment recreated i don't want to call `checkForSellLock` twice `needCheckForSellLock` is control it